package.json

```diff
@@ -6,7 +6,9 @@
   "bugs": {
     "url": "https://github.com/mozilla/tab-split-2/issues"
   },
-  "dependencies": {},
+  "dependencies": {
+    "testpilot-ga": "^0.3.0"
```

Comment:
Is it a devDependency or a dependency? Since this isn't really a Node package I can't tell ;)

Reply:
I think it is a devDependency, since it only gets used at build time.